lib/assessment.md

```diff
@@ -41,7 +50,10 @@ What selectors could we write to reference each of the following categories of e
 * Any `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
-
+.big {}
+#contentPane {}
+.important .red {}
+ .div.p {}
```

Comment:
No, sorry. Use `div > p` to indicate that we want `p`s that are child elements (as opposed to descendants) of `div`s.